src/functions-reference/complex_matrix_operations.Rmd

```diff
@@ -1441,6 +1472,28 @@ Compute the unitary matrix of the complex Schur decomposition of `A`.
 `r since("2.31")`
 
 
+<!-- tuple(complex_matrix, complex_matrix); complex_schur_decompose; (matrix A); -->
+\index{{\tt \bfseries complex\_schur\_decompose }!{\tt (matrix A): tuple(complex\_matrix, complex\_matrix)}|hyperpage}
```

Comment:
Is there a reason this isn't just a promotion of the complex case?  Is there a different algorithm from Eigen?

Reply:
The C++ implementation in `stan::math` is happy with either complex matrices or real ones. Eigen internally does template on whether or not the scalars are complex to optimize which routine is used.

It's important we expose the extra signature in the compiler to avoid the unnecessary promotion, but we don't need to document it if you'd prefer we didn't

Reply:
In that case, I'd suggest adding to the doc a note that it's more efficient for real-valued inputs.